eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/io/DbsnpVariantsWriterTest.java

```diff
@@ -914,6 +954,374 @@ public void declusterRs638662487() throws Exception {
+        wrapper4.setClusteredVariant(clusteredVariantEntity4);
+        wrapper4.setSubmittedVariants(Collections.singletonList(submittedVariantEntity4));
+
+        // when
+        dbsnpVariantsWriter.write(Arrays.asList(wrapper, wrapper2, wrapper3, wrapper4));
+
+        // then
+        assertClusteredVariantStored(1, wrapper);
+        DbsnpSubmittedVariantEntity expectedSubmittedVariantEntity2 = changeRS(submittedVariantEntity2,
+                                                                               clusteredVariantEntity1.getAccession());
+        DbsnpSubmittedVariantEntity expectedSubmittedVariantEntity3 = changeRS(submittedVariantEntity3,
+                                                                               clusteredVariantEntity1.getAccession());
+        assertSubmittedVariantsStored(2, submittedVariantEntity1, expectedSubmittedVariantEntity3);
+        assertSubmittedVariantsHaveActiveClusteredVariantsAccession(wrapper.getClusteredVariant().getAccession(),
+                                                                    expectedSubmittedVariantEntity2,
+                                                                    expectedSubmittedVariantEntity3);
+        assertSubmittedUpdateOperationsHaveClusteredVariantAccession(5, 1, clusteredVariantAccession2);
+        assertSubmittedUpdateOperationsHaveClusteredVariantAccession(5, 2, clusteredVariantAccession3);
+        assertSubmittedVariantMergeOperationStored(5, 1, submittedVariantEntity1);
+        assertSubmittedVariantMergeOperationStored(5, 1, submittedVariantEntity3);
+        // could be 3 operations total? 2 copies for clusteredVariantEntity3
+
+        assertEquals(0, mongoTemplate.count(new Query(), DBSNP_CLUSTERED_VARIANT_DECLUSTERED_COLLECTION_NAME));
+    }
+
+    /**
+     * Clustered variant rs136611820 was merged into several other clustered variants with the same hash: [42568024,
+     * 42568025]
+     * <p>
+     * This happened because rs136611820 had several locations. One matched the location of rs42568024 and other matched
+     * the location of rs42568025. This makes it harder to decide which RS should be the active one and what to do with
+     * the other RSs.
+     * <p>
+     * The desired result is that an RS can be merged several times into other RSs if they all have the same hash, but
+     * in the main collection only one of those will be present.
+     * <p>
+     * This test is similar to the previous one, but closer to the real case, because this one involves also the
+     * declusterings.
+     */
+    @Test
+    public void rs136611820() throws Exception {
+        // given
+        DbsnpVariantsWrapper wrapper1 = new DbsnpVariantsWrapper();
+        DbsnpVariantsWrapper wrapper2 = new DbsnpVariantsWrapper();
+        DbsnpVariantsWrapper wrapper3 = new DbsnpVariantsWrapper();
+        DbsnpVariantsWrapper wrapper4 = new DbsnpVariantsWrapper();
+
+        setupRs136611820(wrapper1, wrapper2, wrapper3, wrapper4);
```

Comment:
Could you please ensure that the contig start positions do not differ for the ss IDs during the setup? While I understand that they are irrelevant, they are an unneeded distraction when debugging.